specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5247 @@
+          ],
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The properties for update Quota response."
+    },
+    "ResourceName": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the resource."
+        },
+        "localizedValue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized name of the resource."
+        }
+      },
+      "description": "The Resource Name."
+    },
+    "ResourceQuota": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the resource ID."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the resource type."
+        },
+        "name": {
+          "$ref": "#/definitions/ResourceName",
+          "description": "Name of the resource.",
+          "readOnly": true
+        },
+        "limit": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "title": "Limit.",
+          "description": "The maximum permitted quota of the resource."
+        },
+        "unit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "An enum describing the unit of quota measurement.",
+          "enum": [
+            "Count"
+          ],
+          "x-ms-enum": {
+            "name": "QuotaUnit",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The quota assigned to a resource."
+    },
+    "ListWorkspaceQuotas": {
+      "properties": {
+        "value": {
+          "description": "The list of Workspace Quotas by VM Family",
+          "items": {
+            "$ref": "#/definitions/ResourceQuota"
+          },
+          "readOnly": true,
+          "type": "array"
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page of workspace quota information by VM Family. Call ListNext() with this to fetch the next page of Workspace Quota information.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "description": "The List WorkspaceQuotasByVMFamily operation response."
+    },
+    "Identity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
+            "SystemAssigned",
+            "SystemAssigned,UserAssigned"
```

Comment:
You accept System+User but not just User?

Reply:
updated.